integration/containerd/cri/container_restart_test.go.patch

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+// Copyright (c) 2018 Intel Corporation
```

Comment:
nit: you could add yourself/org to the copyright header maybe if you wanted.
And trim the apache license header down to just the SPDX style one, to match the rest of the kata code.

Reply:
Thanks ! Updated to be consistent with the cri patch :)